packages/contracts-bedrock/.gas-snapshot

```diff
@@ -1,3 +1,9 @@
+L1CrossDomainMessenger_GasBenchMark:test_L1MessengerSendMessage_benchmark() (gas: 160692)
+L1StandardBridge_Deposit_GasBenchMark:test_depositERC20_benchmark() (gas: 251966)
+L1StandardBridge_Deposit_GasBenchMark:test_depositETH_benchmark() (gas: 251968)
```

Comment:
Pretty big number. Though, per the trace, I think `240024` would be more accurate:

```
[PASS] test_depositETH_benchmark() (gas: 251968)
Traces:
  [251968] L1StandardBridge_Deposit_GasBenchMark::test_depositETH_benchmark()
    ├─ [240024] L1StandardBridge_Proxy::depositETH{value: 500}(50000, 0x)
```

Still, relative to our [current benchmarks](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/test/contracts/L1/messaging/deposit.gas.spec.ts#L124), this looks rough. I haven't yet looked closely to see where the costs are coming from.

Reply:
Here's the full trace, `OptimismPortal::depositTransaction` is surprisingly expensive!

```
Traces:
  [253156] GasBenchMark_L1StandardBridge_Deposit::test_depositETH_benchmark()
    ├─ [241194] L1StandardBridge::depositETH{value: 500}(50000, 0xff)
    │   ├─ [0] bob::b7947262() [staticcall]
    │   │   └─ ← ()
    │   ├─ [230977] L1StandardBridge::depositETH(50000, 0xff) [delegatecall]
    │   │   ├─ emit ETHDepositInitiated(_from: alice: [0x0000000000000000000000000000000000000080], _to: alice: [0x0000000000000000000000000000000000000080], _amount: 500, _data: 0xff)
    │   │   ├─ emit ETHBridgeInitiated(_from: alice: [0x0000000000000000000000000000000000000080], _to: alice: [0x0000000000000000000000000000000000000080], _amount: 500, _data: 0xff)
    │   │   ├─ [209057] Lib_ResolvedDelegateProxy::sendMessage{value: 500}(L2StandardBridge: [0x4200000000000000000000000000000000000010], 0x1635f5fd0000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000001f400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000001ff00000000000000000000000000000000000000000000000000000000000000, 50000)
    │   │   │   ├─ [3213] Lib_AddressManager::getAddress("OVM_L1CrossDomainMessenger") [staticcall]
    │   │   │   │   └─ ← L1CrossDomainMessenger: [0xf5a2fe45f4f1308502b1c136b9ef8af136141382]
    │   │   │   ├─ [195429] L1CrossDomainMessenger::sendMessage(L2StandardBridge: [0x4200000000000000000000000000000000000010], 0x1635f5fd0000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000001f400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000001ff00000000000000000000000000000000000000000000000000000000000000, 50000) [delegatecall]
    │   │   │   │   ├─ [151459] OptimismPortal::depositTransaction{value: 500}(L2CrossDomainMessenger: [0x4200000000000000000000000000000000000007], 500, 150196, false, 0xd764ad0b00010000000000000000000000000000000000000000000000000000000000000000000000000000000000003a1148fe01e3c4721d93fe8a36c2b5c29109b6ae000000000000000000000000420000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000001f4000000000000000000000000000000000000000000000000000000000000c35000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000c41635f5fd0000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000001f400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000001ff0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000)
    │   │   │   │   │   ├─ emit TransactionDeposited(from: L1CrossDomainMessenger_aliased: [0x53aa7ac9251e5bb0a61f4ff790e5b991ea080eac], to: L2CrossDomainMessenger: [0x4200000000000000000000000000000000000007], mint: 500, value: 500, gasLimit: 150196, isCreation: false, data: 0xd764ad0b00010000000000000000000000000000000000000000000000000000000000000000000000000000000000003a1148fe01e3c4721d93fe8a36c2b5c29109b6ae000000000000000000000000420000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000001f4000000000000000000000000000000000000000000000000000000000000c35000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000c41635f5fd0000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000001f400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000001ff0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000)
    │   │   │   │   │   └─ ← ()
    │   │   │   │   ├─ emit SentMessage(target: L2StandardBridge: [0x4200000000000000000000000000000000000010], sender: L1StandardBridge: [0x3a1148fe01e3c4721d93fe8a36c2b5c29109b6ae], message: 0x1635f5fd0000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000001f400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000001ff00000000000000000000000000000000000000000000000000000000000000, messageNonce: 1766847064778384329583297500742918515827483896875618958121606201292619776, gasLimit: 50000)
    │   │   │   │   └─ ← ()
    │   │   │   └─ ← ()
    │   │   └─ ← ()
    │   └─ ← ()
    └─ ← ()
```

Reply:
I bet this is because of the gas burn. When the base fee is low (which it probably is within the forge VM), we end up burning much more gas.